### ISSUE TITLE
[FSR] Empty state for when there are no debts

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#20249]
## Issue Description
Currently, a Veteran can technically access the FSR form when they do not have an active debt. Investigate our options and define the design direction for this.

- Does the FSR need to be available for authenticated Veterans with no debts?
  - Is there any way to prevent authenticated Veterans with no debts from seeing them, technically?
- Should this design mimic the unauthenticated debt selection screen?

---
## Tasks
- [x] Determine design direction for Authenticated FSRs with no active debts, document in GH & comment in this ticket
- [x] Create a ticket for frontend

## Acceptance Criteria
- [x] Direction defined in GH and commented in this ticket
- [x] Ticket created for front end and linked

**Comment by rileyorr** [comment]
I chatted with Craig about when the API call is made to check for debts on an authenticated user's file, and he confirmed that the API call occurs after the intro screen once the Veteran moves into the form. I've added [a mockup of the empty state that informs the Veteran that they don't have any debt related to VA benefits and directs them to contact DMC if this is an error](https://preview.uxpin.com/803dc24213cf723bfce6c5e3eaef5a2c3f3e344b#/pages/137187972/specification/no-panels?mode=i). 

The pros and cons were weighed to allow the Veteran to submit the FSR anyway with the unauthenticated debt selection flow; when considering the user's perspective they likely wouldn't proceed to complete the form. If they do have debt then they'd probably want more information before submitting all their financial information. 

[I've added a frontend ticket for this work](https://github.com/department-of-veterans-affairs/va.gov-team/issues/20773).
